VoteCounter/src/GUI.cs

```diff
@@ -29,8 +29,48 @@ private void btnGenerateVoteCount_Click(object sender, EventArgs e)
             {
                 PostList postList = new PostList(url, startingPostNumber, endingPostNumber);
 
+                //make a List
+                List<Player> players = new List<Player>();
+
+                //set the players
+                foreach (DataGridViewRow row in listPlayers.Rows)
+                {
+                    List<string> nicks = new List<string>();
+                    string name = "";
+                    for (int i = 0; i < listPlayers.Columns.Count; i++)
+                    {
+                        //if under column Names
+                        if (listPlayers.Columns[i].Name.Equals("Name")
+                            && row.Cells[i].Value != null)
+                            name = row.Cells[i].Value.ToString();
+
+                        //if under column Nicknames
+                        if (listPlayers.Columns[i].Name.Equals("Nicknames")
+                            && row.Cells[i].Value != null)
+                        {
+                            if (row.Cells[i].Value.ToString().Length > 0)
+                            {
+                                //parse the string, seperate each name, then put it in the List
+                                string[] words = row.Cells[i].Value.ToString().Split(',');
+                                //then add them
+                                foreach (string s in words)
+                                {
+                                    nicks.Add(s);
+                                }
+                                //its adding a blank to the end, so I'll just remove that manually
+                                if (nicks.Contains(""))
```

Comment:
We could either do it this way or try to remove the final comma before we parse it. Do you have a preference?